services/m3dbnode/config/bootstrap.go

```diff
@@ -53,6 +53,10 @@ type BootstrapConfiguration struct {
 
 	// Peers bootstrapper configuration.
 	Peers *BootstrapPeersConfiguration `yaml:"peers"`
+
+	// CacheSeriesMetadata determines whether individual bootstrappers cache
+	// series metadata across all calls (namespaces / shards / blocks).
+	CacheSeriesMetadata *bool `yaml:"cacheSeriesMetadata"`
```

Comment:
Hm should this renamed to be only for the commit log bootstrapper? I don't think we'd do the caching for data in peers/fs as things stand now.

Reply:
Made this change (from being at the commit log bootstrapper level) at @robskillington request. Seemed reasonable so that if we implement those we don't need to add additional config

Reply:
(I think we will want to do it at the peer level at least)